pac/pipelines/gitops-pull-request-rhtap.yaml

```diff
@@ -27,11 +27,11 @@ spec:
     description: The public key that EC should use to verify signatures
     name: ec-public-key
     type: string
-  - default: http://rekor-server.rhtap-tas.svc
+  - default: http://rekor-server.tssc-tas.svc
```

Comment:
Needs to be updated here first: https://github.com/konflux-ci/build-definitions/blob/main/pipelines/gitops-pull-request-rhtap/gitops-pull-request.yaml

Reply:
@lcarva Done, see https://github.com/konflux-ci/build-definitions/pull/2311